packages/framework/src/Framework/Features/Navigation/NavItem.php

```diff
@@ -14,6 +14,8 @@
 /**
  * Abstraction for a navigation menu item. Used by the MainNavigationMenu and DocumentationSidebar classes.
  *
+ * @FIXME Before merge: Add release notes from https://github.com/hydephp/develop/pull/1539
```

Comment:
Before merge: Add release notes from https://github.com/hydephp/develop/pull/1539

Reply:
Resolved in https://github.com/hydephp/develop/commit/c133c82ca93a12b27954aff50a69970072edf6d0